app/frontend/src/features/profile/view/leaveReference/ReferenceForm.tsx

```diff
@@ -105,19 +104,19 @@ export default function ReferenceForm({ user }: ReferenceFormProps) {
       referenceData={referenceData}
       setReferenceValues={setReferenceValues}
     />
-  ) : step === "rating" ? (
+  ) : step === steps[1] ? (
     <Rating
       user={user}
       referenceData={referenceData}
       setReferenceValues={setReferenceValues}
     />
-  ) : step === "reference" ? (
+  ) : step === steps[2] ? (
     <Text
       user={user}
       referenceData={referenceData}
       setReferenceValues={setReferenceValues}
     />
-  ) : step === "submit" ? (
+  ) : step === steps[3] ? (
```

Comment:
Hmm I'd keep it as before tbh, this is now making the code more obscure to read imo. Since these steps are fixed, it's better if you pull these out as constants so it reads more similar to before with `step === REFERENCE` etc